### ISSUE TITLE
Opening sidebar links in new window 

### ISSUE THREAD

**Issue by sover02** [creativetimofficial/ct-vue-black-dashboard-pro#25]
Clicking a link in the sidebar opens the page correctly. Example:
https://demos.creative-tim.com/vue-black-dashboard-pro/#/widgets

Clicking to open it in a new window attempts to send the user to the base root + `widgets`:
https://demos.creative-tim.com/widgets

This leads to a 404.

I've tried editing the `<sidebar-item>` `path` attributes like so:
```
				<sidebar-item
					:link="{
						name: 'USERS',
						icon: 'tim-icons icon-single-02',
						path: '#/users',
					}"
				></sidebar-item>
```
and 
```
				<sidebar-item
					:link="{
						name: 'USERS',
						icon: 'tim-icons icon-single-02',
						path: '#/users',
					}"
				></sidebar-item>
```
but both results in chained urls such as `<site>/#/widgets/#/widgets`

Any simple fix?

**Comment by sover02** [comment]
Another attempt.
Changing `path` to:
```path: './#/workers',```
Fixes the new window, but breaks the regular navigation.

**Comment by sover02** [comment]
I solved this by changing 

```
:href="'/#'+link.path"
```
to
```
:href="link.path"
```
In `SidebarItem.vue`.  Is there a better way?

**Comment by dragosct10** [comment]
Hi, @sover02! Thanks for using our products. The /# should not be there so yes this is a good workaround. 

Regards,
Dragos

**Comment by sover02** [comment]
Sorry, I miscommunicated – it was the other way around - I had to /add/ the `/#` to `:href`.

**Comment by dragosct10** [comment]
Hi! Now the link looks like this http://localhost:8080/#/dashboard#/widgets?

**Comment by sover02** [comment]
Once I made the above change, I was able to set the path in attributes as such:
```
				<sidebar-item
					:link="{
						name: 'USERS',
						icon: 'tim-icons icon-single-02',
						path: 'users',
					}"
				></sidebar-item>
```

I think setting the path to include a `/` was telling the browser to append to or skip the basepath.

Sorry for the confusion, been bouncing around quite a bit.

**Comment by dragosct10** [comment]
Ok! Now everything is fine, right?

**Comment by sover02** [comment]
Yep!  I'm not super strong on the Vue side, but I was hoping there was a more elegant place to add the literal string.

I was wondering if it would be worth implementing in upcoming versions of the template.  You can close the issue if not.

**Comment by dragosct10** [comment]
Yes, we can take a closer look to see a better way to the literal string in an upcoming update. 

Regards,
Dragos
